pageserver/src/basebackup.rs

```diff
@@ -413,6 +413,10 @@ where
             let mut content = Vec::with_capacity(n_origins * 16 + 8);
             content.extend_from_slice(&pg_constants::REPLICATION_STATE_MAGIC.to_le_bytes());
             for (origin_id, origin_lsn) in repl_origins {
+                info!(
```

Comment:
could you please wrap all of these log lines with `if cfg!(feature = "testing")`, so that they only get logged in regression tests?

Reply:
Sorry, this is temporary PR - just for investigation.
I am not going to merge it.